### PR TITLE
Adds packer build for server with VM Single node in vultr.com marketplace

### PR DESCRIPTION
**Pull request by denisgolius**
This PR will add packer config and helper scripts to build a virtual server with VM Single node for https://www.vultr.com/marketplace/categories/#monitoring. It also adds a Makefile and RELEASE_GUIDE.md.

### CODE REVIEW

File: deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+In a web browser, you can view:
+ * The VictoriaMetrics Quickstart guide: https://kutt.it/1click-quickstart
+
+On the server:
+  * The default VictoriaMetrics root is located at /var/lib/victoria-metrics-data
+  * VictoriaMetrics is running on ports: 8428, 8089, 4242, 2003 and they are bound to the local interface.
+
+********************************************************************************
+  # This image includes version v1.81.2 of VictoriaMetrics.
```
Comment (zekker6) [inline deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click:22]: Should version here be templated from build variable?

Comment (denisgolius) [reply deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click]: yep, fixed

File: deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+In a web browser, you can view:
+ * The VictoriaMetrics Quickstart guide: https://kutt.it/1click-quickstart
+
+On the server:
+  * The default VictoriaMetrics root is located at /var/lib/victoria-metrics-data
+  * VictoriaMetrics is running on ports: 8428, 8089, 4242, 2003 and they are bound to the local interface.
+
+********************************************************************************
+  # This image includes version VM_VERSION of VictoriaMetrics.
```
Comment (zekker6) [inline deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click:22]:
```suggestion
  # This image includes version $VM_VERSION of VictoriaMetrics.
```

File: deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click.tpl [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+In a web browser, you can view:
+ * The VictoriaMetrics Quickstart guide: https://kutt.it/1click-quickstart
+
+On the server:
+  * The default VictoriaMetrics root is located at /var/lib/victoria-metrics-data
+  * VictoriaMetrics is running on ports: 8428, 8089, 4242, 2003 and they are bound to the local interface.
+
+********************************************************************************
+  # This image includes version VM_VERSION of VictoriaMetrics.
```
Comment (zekker6) [inline deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click.tpl:22]:
```suggestion
  # This image includes version $VM_VERSION of VictoriaMetrics.
```

File: deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click (hunk 2 above)
Comment (denisgolius) [reply deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click:22]: There is a default template file `99-one-click.tpl`, which is copied to 99-one-click where the `VM_VERSION` is replaced with the version passed to make - see https://github.com/VictoriaMetrics/VictoriaMetrics/pull/3142/files/36a6bdbc5f8e6590e4f3002b5198535ae5f45a71#diff-454031dadedbca78b6d08f2e42cba83bbcc8999ac99778e38dca05e6b19cd5a3

Comment (zekker6) [reply deployment/marketplace/vultr/victoriametrics-single/etc/update-motd.d/99-one-click:22]: Ah, ok, got it.

**Review by zekker6 — APPROVED** [review]

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@denisgolius , thanks for the pull request!

**Pull request merged**